### DISCUSSION TITLE
Meeting Type Families and Friends hyper link text extends under map

### DISCUSSION THREAD

**Discussion by RockfaceIT** [code4recovery/12-step-meeting-list#1364] [Q&A]
Hello!
I am using Elementor Pro with a Hello Elementor Child Theme.  I do not have any special CSS, code or anything.  I tried turning off all plugins to see if that would fix this issue - it did not fix it.

If I change the theme to twenty twenty four the error sort of goes away but still does not look ideal.

If you goto any meeting with the Meeting Type 'Friend and Families' or 'Friends, Families and Observers'.  There is a SVG '?' hyperlink after 'Friends and Families'.  If the '?' is clicked then a whole paragraph shows up below.  The line proceeds outside of the Meeting Information box and under the map area.  

Please see the screenshot...Let me know if you need anything else.

<img width="1201" alt="Screen Shot 2024-02-19 at 10 30 43 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/37557944/89c1236e-ca00-4935-8f5f-1c3393320f17">

Example LInk:
https://alanonventura.rockfaceit.com/meetings/?meeting=wednesday-agoura-morning-meeting

<img width="1219" alt="Screen Shot 2024-02-19 at 10 37 42 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/37557944/76b5dd94-9fa2-4eb9-8acc-54a88511e82a">

<img width="478" alt="Screen Shot 2024-02-19 at 9 29 55 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/37557944/d157224e-a0ae-4568-a1b5-3fe7b0f9e5c3">

Thank you!

**Comment by joshreisner** [comment]
ok i've [updated TSML UI now](https://github.com/code4recovery/tsml-ui/pull/363) and it should be fixed if you refresh
